bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
I believe this tag *might* still be needed, as it is also used for the other strings (further down the template) that occur within the modal. Removing it here would expose the modal link, regardless to if the modal is translated.

@flodolo - if enough locales have translated all the modal strings, I agree this can be removed.